components/RichText.tsx

```diff
@@ -0,0 +1,95 @@
+import { documentToReactComponents } from '@contentful/rich-text-react-renderer';
+import { BLOCKS, MARKS, INLINES } from '@contentful/rich-text-types';
+import React from 'react';
+import styles from 'styles/RichText.module.scss';
+
+type Theme = 'LIGHT' | 'DARK' | 'INDIGO';
+
+type RichTextProps = {
+    theme: Theme;
+    content: any;
+};
+
+const renderParagraph = (theme: Theme, children: React.ReactNode) => {
+    switch (theme) {
+        case 'LIGHT':
+            return <p className={styles.paragraphLight}>{children}</p>;
+        case 'DARK':
+            return <p className={styles.paragraphDark}>{children}</p>;
+        case 'INDIGO':
+            return <p className={styles.paragraphIndigo}>{children}</p>;
+    }
+};
+
+const renderLink = (theme: Theme, node: any, children: React.ReactNode) => {
+    switch (theme) {
```

Comment:
**Praise:** I really like your use of switch statements, you've actually influenced me to use them more.